## notebook for script p40.py
* version 1


## G. Papalambrou, 8/4/2020

In [10]:
def test(x, y):
    """ this is a help docstring for my function text(). BLA BLA
    BL BLAAAA """
    a=x/y
    b=x%y
    return a,b

In [3]:
test(5,3)


(1, 2)

### results from the above call:

Build the `tf.keras.Sequential` model by stacking layers. Choose an optimizer and loss function for training:

*This notebook contains an excerpt from the [Python Data Science Handbook](http://shop.oreilly.com/product/0636920034919.do) by Jake VanderPlas; the content is available [on GitHub](https://github.com/jakevdp/PythonDataScienceHandbook).*


In [4]:
c, d = test(5,2)


In [5]:
print(c)


2


In [6]:
print(d)


1


* hello again

In [11]:
help(test)

Help on function test in module __main__:

test(x, y)
    this is a help docstring for my function text(). BLA BLA
    BL BLAAAA



* idea 1: how to conver two numbers 
* idea 2: transmit power

In [12]:
%%time
for i in range (0,10):
    print("xxx-{}".format(i))

xxx-0
xxx-1
xxx-2
xxx-3
xxx-4
xxx-5
xxx-6
xxx-7
xxx-8
xxx-9
CPU times: user 155 µs, sys: 46 µs, total: 201 µs
Wall time: 204 µs


In [13]:
%%bash
ls -all

total 128
drwxrwxrwx 5 gpapalambrou gpapalambrou 4096 Απρ   9 18:33 .
drwxrwxrwx 7 gpapalambrou gpapalambrou 4096 Απρ   9 17:41 ..
-rwxrwxrwx 1 gpapalambrou gpapalambrou  627 Απρ   2 22:07 3_1.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  570 Φεβ  28  2019 3_1.py~
-rwxr-xr-x 1 gpapalambrou gpapalambrou  345 Απρ   2 22:05 3_2.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  348 Μαρ  20  2019 atom.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  210 Απρ   1 18:49 break.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou 1031 Μαρ  13  2019 easyMath1.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  419 Μαρ  13  2019 ex_functions.py
-rw-r--r-- 1 gpapalambrou gpapalambrou   60 Απρ   1 20:11 guis.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou   93 Φεβ  28  2019 hello1.py
drwxrwxrwx 2 gpapalambrou gpapalambrou 4096 Μαρ  21  2019 .idea
-rwxrwxrwx 1 gpapalambrou gpapalambrou   98 Μαρ  13  2019 imptee.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  421 Μαρ  13  2019 imptee.pyc
-rwxrwxrwx 1 gpapalambrou gpapalambrou  139 Μαρ  13  2019 im

In [16]:
%%bash
ls -all

total 140
drwxrwxrwx 5 gpapalambrou gpapalambrou 4096 Απρ   9 19:19 .
drwxrwxrwx 7 gpapalambrou gpapalambrou 4096 Απρ   9 17:41 ..
-rwxrwxrwx 1 gpapalambrou gpapalambrou  627 Απρ   2 22:07 3_1.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  570 Φεβ  28  2019 3_1.py~
-rwxr-xr-x 1 gpapalambrou gpapalambrou  345 Απρ   2 22:05 3_2.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  348 Μαρ  20  2019 atom.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  210 Απρ   1 18:49 break.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou 1031 Μαρ  13  2019 easyMath1.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  419 Μαρ  13  2019 ex_functions.py
-rw-r--r-- 1 gpapalambrou gpapalambrou   60 Απρ   1 20:11 guis.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou   93 Φεβ  28  2019 hello1.py
drwxrwxrwx 2 gpapalambrou gpapalambrou 4096 Μαρ  21  2019 .idea
-rwxrwxrwx 1 gpapalambrou gpapalambrou   98 Μαρ  13  2019 imptee.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  421 Μαρ  13  2019 imptee.pyc
-rwxrwxrwx 1 gpapalambrou gpapalambrou  139 Μαρ  13  2019 im

In [12]:
%%bash
date


Παρ 10 Απρ 2020 09:19:54 πμ EEST


In [13]:
%%bash
ls -all

total 172
drwxrwxrwx 5 gpapalambrou gpapalambrou  4096 Απρ  10 09:19 .
drwxrwxrwx 7 gpapalambrou gpapalambrou  4096 Απρ   9 20:02 ..
-rw-rw-r-- 1 gpapalambrou gpapalambrou 25475 Απρ   9 19:58 2D_test_matplot.ipynb
-rwxrwxrwx 1 gpapalambrou gpapalambrou   627 Απρ   2 22:07 3_1.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou   570 Φεβ  28  2019 3_1.py~
-rwxr-xr-x 1 gpapalambrou gpapalambrou   345 Απρ   2 22:05 3_2.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou   348 Μαρ  20  2019 atom.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou   210 Απρ   1 18:49 break.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou  1031 Μαρ  13  2019 easyMath1.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou   419 Μαρ  13  2019 ex_functions.py
-rw-r--r-- 1 gpapalambrou gpapalambrou    60 Απρ   1 20:11 guis.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou    93 Φεβ  28  2019 hello1.py
drwxrwxrwx 2 gpapalambrou gpapalambrou  4096 Μαρ  21  2019 .idea
-rwxrwxrwx 1 gpapalambrou gpapalambrou    98 Μαρ  13  2019 imptee.py
-rwxrwxrwx 1 gpapalambrou gpapalamb

total 172
drwxrwxrwx 5 gpapalambrou gpapalambrou  4096 Απρ  10 09:19 .
drwxrwxrwx 7 gpapalambrou gpapalambrou  4096 Απρ   9 20:02 ..
-rw-rw-r-- 1 gpapalambrou gpapalambrou 25475 Απρ   9 19:58 2D_test_matplot.ipynb
-rwxrwxrwx 1 gpapalambrou gpapalambrou   627 Απρ   2 22:07 3_1.py
-rwxrwxrwx 1 gpapalambrou gpapalambrou   570 Φεβ  28  2019 3_1.py~
-rwxr-xr-x 1 gpapalambrou gpapalambrou   345 Απρ   2 22:05 3_2.py

* skfjsdkj;kds 

In [32]:
def intersect(seq1, seq2):
    res = []
    for x in seq1:
        if x in seq2:
            res.append(x)
    return res

In [33]:
s1 = "PSAM"

In [34]:
s2 = "SCAM"

In [30]:
intersect(s1, s2)

['S', 'A', 'M']

In [35]:
x = intersect([1, 2, 3], [1, 4])

In [14]:
def times(x, y):
    return x * y

In [15]:
times(2, 4)

8

In [36]:
x


[1]